converter/src/main/java/org/dswarm/converter/pipe/timing/XmlTimer.java

```diff
@@ -0,0 +1,280 @@
+/**
+ * Copyright (C) 2013 – 2015 SLUB Dresden & Avantgarde Labs GmbH (<code@dswarm.org>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dswarm.converter.pipe.timing;
+
+import java.io.IOException;
+import java.util.Deque;
+import java.util.LinkedList;
+
+import com.codahale.metrics.MetricRegistry;
+import com.google.inject.Inject;
+import com.google.inject.assistedinject.Assisted;
+import com.google.inject.name.Named;
+import org.culturegraph.mf.framework.ObjectReceiver;
+import org.culturegraph.mf.framework.XmlPipe;
+import org.culturegraph.mf.framework.XmlReceiver;
+import org.xml.sax.Attributes;
+import org.xml.sax.InputSource;
+import org.xml.sax.Locator;
+import org.xml.sax.SAXException;
+import org.xml.sax.SAXParseException;
+
+
+public final class XmlTimer<R> extends TimerBased<XmlPipe<ObjectReceiver<R>>> implements XmlReceiver {
```

Comment:
I'm not sure, whether we need to measure the xml processing at that level of detail. Maybe the number of processed entities and literals -> i.e. the generated statements (hence number of statements) and records are enough, or?
Doesn't this downgrade the performance otherwise?